docs/src/architecture/domain-model.md

```diff
@@ -0,0 +1,214 @@
+# thin-edge Domain Model
+
+The following section introduces the **thin-edge** domain model.
+
+The **thin-edge** domain model explains details of target domains **thin-edge** is primarily designed for.
```

Comment:
In general, I like https://cumulocity.com/guides/concepts/domain-model/ more: More to the point.

IMHO this section introduces quite some terms with weak semantics, and the terms are not relevant for the audience to understand thin-edge domain model ("target domain", Entity, Aspects, target domain expert).

My suggestion: Drop

Reply:
That section was dropped now.